bundles/EcommerceFrameworkBundle/PricingManager/PricingManager.php

```diff
@@ -228,7 +231,7 @@ public function getValidRules()
     public function getEnvironment()
     {
         /** @var AttributeBagInterface $sessionBag */
-        $sessionBag = $this->session->getBag(SessionConfigurator::ATTRIBUTE_BAG_PRICING_ENVIRONMENT);
+        $sessionBag = $this->session->getBag(SessionBagListener::ATTRIBUTE_BAG_PRICING_ENVIRONMENT);
 
         $class = $this->options['environment_class'];
 
```

Comment:
we also should add deprecation notices to the `get/setSession` from `Pimcore\Bundle\EcommerceFrameworkBundle\PricingManager\EnvironmentInterface`

Reply:
done